iina/Utility.swift

```diff
@@ -70,12 +71,24 @@ class Utility {
       alert.informativeText = String(format: format)
     }
 
+    if let suppressionKey = suppressionKey {
```

Comment:
Can we move the check right after `let alert` line?

Reply:
Good catch. Will move.

Reply:
On the resetting of suppressed alerts, I will work on adding a new section to Preferences/Utilities that looks like:

---

#### Restore Alerts

The button below will restore all alerts that have been suppressed using the "Do not show this message again" checkbox.
`Restore Suppressed Alerts`

Reply:
The latest commit adds a `Restore Suppressed Alerts` button in the `Utilities` section of `Preferences` giving users the ability to easily change IINA back to showing alerts. Remember I am a total newbie when it comes to GUI programming. Be on the lookout for newbie mistakes.

The `Utilities` section of `Preferences` now looks like:
![issue-3842-util](https://user-images.githubusercontent.com/86170219/186980673-6e5eb660-a398-47af-bf9f-df13705f8c98.jpg)

Clicking the button asks for confirmation:
![issue-3842-util-alert](https://user-images.githubusercontent.com/86170219/186980684-f4960bea-a20a-4dcc-aac2-0aa8b59f112a.jpg)

If `OK` is clicked `Restored.` is shown similar to other buttons in `Utilities`:
![issue-3842-restored](https://user-images.githubusercontent.com/86170219/186980698-d46ebee8-ac96-487c-a858-13e31f317995.jpg)